### PR TITLE
chore: merge with wiki

### PR DESCRIPTION
**Pull request by hoffa**
Merging information from the [plugin page](https://plugins.jenkins.io/aws-sam/), as the page itself can't be edited (see sidebar for more info). Will have to mirror GitHub README.

- [ ] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your master branch!
- [ ] Ensure that the pull request title represents the desired changelog entry
- [ ] Please describe what you did
- [ ] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,69 @@
...
+## Features
 
-* [samDeploy](#samdeploy)
+> If you wish to use the AWS SAM CLI directly, you can use [AWS SAM build images](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-image-repositories.html) with Pipeline's [built-in Docker support](https://www.jenkins.io/doc/book/pipeline/docker/).
 
-If you wish to use the AWS SAM CLI directly, you can use [AWS SAM build images](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-image-repositories.html) with Pipeline's [built-in Docker support](https://www.jenkins.io/doc/book/pipeline/docker/).
+- Custom build step
+- [`samDeploy`](https://www.jenkins.io/doc/pipeline/steps/aws-sam/) Pipeline step
+- Deploy SAM projects to AWS
+- Package SAM projects to a S3
```
Comment (aahung) [inline README.md]: We usually say "deploying a SAM application" not a project

> To deploy an AWS SAM application, https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-cli-command-reference-sam-deploy.html

File: README.md [hunk 2]
```diff
@@ -1,39 +1,69 @@
...
+  - Use specific or auto-generated S3 bucket (if not given, a bucket will be automatically generated in your account for you)
```
Comment (aahung) [inline README.md]: > auto-generated

auto-created seems better. "Generating" a bucket seems off

File: README.md [hunk 3]
```diff
@@ -1,39 +1,69 @@
...
+- Template parameter support
```
Comment (aahung) [inline README.md]:
```suggestion
- Template parameters support
```

File: README.md [hunk 4]
```diff
@@ -1,39 +1,69 @@
...
+- Template tags support
+- Deploy to specific region
```
Comment (aahung) [inline README.md]:
```suggestion
- Deploy to a specific region
```

or

```suggestion
- Deploy to specific regions
```

**Review by prenx4x — APPROVED** [review]

**Review by aahung — APPROVED** [review]
LGTM!

**Pull request merged**